### PR TITLE
upi/vsphere: create apps dns entry

### PR DESCRIPTION
**Pull request by vrutkovs**
Create wildcard `*.apps` for VSphere UPI so that authentication and console operators would work

### CODE REVIEW

File: upi/vsphere/route53/main.tf [hunk 1]
```diff
@@ -61,3 +61,14 @@ resource "aws_route53_record" "etcd_cluster" {
...
   records = ["${formatlist("0 10 2380 %s", aws_route53_record.etcd_a_nodes.*.fqdn)}"]
 }
+
+resource "aws_route53_record" "ingress" {
+  count = "${var.compute_instance_count == "0" ? "0" : "1"}"
+
+  type           = "A"
+  ttl            = "60"
+  zone_id        = "${aws_route53_zone.cluster.zone_id}"
+  name           = "api.${var.cluster_domain}"
```
Comment (abhinavdahiya) [inline upi/vsphere/route53/main.tf]: shouldn't this be `*.apps` ?

Comment (vrutkovs) [reply upi/vsphere/route53/main.tf]: Fixed

**Pull request merged**